proposals/lip-0045.md

```diff
@@ -24,7 +24,8 @@ Interoperability has the potential to solve the scaling issue of blockchains: In
 
 To achieve interoperability, chains in the Lisk ecosystem (the mainchain and the sidechains participating in interoperability) exchange information via _cross-chain transactions_. As an example, a user can issue a cross-chain transaction to transfer part of its mainchain LSK balance to a sidechain, and subsequently use it to issue one of the sidechain custom commands.
```

Comment:
```suggestion
To achieve interoperability, chains in the Lisk ecosystem (the mainchain and the sidechains participating in interoperability) exchange information via _cross-chain transactions_. As an example, a user can issue a cross-chain transaction to transfer part of their mainchain LSK balance to a sidechain, and subsequently use it to issue one of the sidechain commands.
```

Reply:
I think we do not want to use the term custom as in custom transaction or similarly custom commands any more as conceptually all commands are the on the same level.

Reply:
updated